docs/install.md

```diff
@@ -4,12 +4,16 @@
 
 Hive is published to [OperatorHub](https://operatorhub.io/operator/hive-operator) weekly and this is the best method to install and use Hive if you do not need to build from source.
 
-  1. Create a 'hive' namespace.
-  1. In OpenShift web console navigate to Administrator perspective > Operators > OperatorHub.
-  1. Search for “hive” and select the OpenShift Hive operator.
-     1. Select the “alpha” update channel, install to a specific namespace, select the “hive” namespace just created, approval strategy automatic, and press Install.
-     1. You should now have a hive-operator pod running in the hive namespace.
-  1. Create a HiveConfig to trigger the the actual deployment of Hive. (can be done via the web UI with a couple clicks, or with oc apply)
+  1. Create a 'hive' namespace doing
+  `$ oc new-project hive`
+
+  2. Install the Hive Operator:
```

Comment:
These bullets should be prefixed with `1.` s.t. the number enumeration is done by markdown. The newline preceding this line will need to be removed for this to work.

Reply:
Done. Thanks!